integration/spark2/src/main/java/org/apache/carbondata/spark/readsupport/SparkRowReadSupportImpl.java

```diff
@@ -36,7 +36,7 @@
     isMeasure = new boolean[carbonColumns.length];
     dataTypes = new DataType[carbonColumns.length];
     for (int i = 0; i < carbonColumns.length; i++) {
-      isMeasure[i] = !carbonColumns[i].isDimesion();
+      isMeasure[i] = !carbonColumns[i].isDimension();
```

Comment:
actually, the initialization in this class is unnecessary, because `readRow` only return a `new GenericInternalRow(data)`, so I will move `isMeasure` variable